airflow/providers/databricks/hooks/databricks_sql.py

```diff
@@ -248,7 +265,22 @@ def run(
                 self.set_autocommit(conn, autocommit)
 
                 with closing(conn.cursor()) as cur:
-                    self._run_command(cur, sql_statement, parameters)  # type: ignore[attr-defined]
+                    t = create_timeout_thread(cur, execution_timeout)
+
+                    # todo; adjust this to make testing easier
+                    try:
+                        self._run_command(cur, sql_statement, parameters)  # type: ignore[attr-defined]
+                    except Exception as e:
+                        if t is None or t.is_alive():
+                            raise AirflowException(f"Error running statement: {sql_statement}. {str(e)}")
+                        else:
+                            raise AirflowException(
```

Comment:
same here

Reply:
I will create an AirflowTimeoutException for this one